frontend/ui/lib/router/router.dart

```diff
@@ -3,13 +3,16 @@ import 'package:tripmeout/pages/create_trip_page.dart';
 import 'package:tripmeout/pages/trip_list_page.dart';
 import 'package:tripmeout/pages/trip_view_page.dart';
 import 'package:tripmeout/services/trip_service.dart';
+import 'package:tripmeout/services/places_services.dart';
 
 class Router {
   static final String tripListRoute = '/trips';
   static final String createTripRoute = '/trips/new';
   static final RegExp tripViewRouteRegExp = RegExp(r'/trips/(?<tripId>[^/]+)$');
 
+
```

Comment:
Nit: not sure if this whitespace is desired per Dart style guide. If the formatter did not specifically insert this, please remove it.